ui/plugins/job_details/main.html

```diff
@@ -5,6 +5,7 @@
       <span ng-if="line.content_type != 'TalosResult'" ng-switch on="line.content_type">
         <a ng-switch-when="link" title="{{line.value}}"
            href="{{line.url}}" target="_blank">{{line.value}}</a>
+        <a title="{{line.value}}" href="http://jgraham.github.io/wptview/#/?urls={{line.url}},{{line.value}}">Open</a>
```

Comment:
I think this is fine as a start. At some point we might want to make to define this in an angular directive so we can re-use this logic as needed.